packages/experimental/Menu/src/MenuItemCheckbox/useMenuItemCheckbox.ts

```diff
@@ -14,16 +14,18 @@ import { useMenuListContext } from '../context/menuListContext';
 const defaultAccessibilityActions = [{ name: 'Toggle' }];
 
 export const useMenuItemCheckbox = (props: MenuItemCheckboxProps): MenuItemCheckboxState => {
-  const { name } = props;
+  const { disabled, name } = props;
```

Comment:
Do we want to also pull out `...rest`  and pass it into a root slot somewhere below?

Reply:
That should be taken care of in useMenuCheckboxInteraction on line 50.